bundles/org.openhab.binding.caddx/src/main/resources/OH-INF/thing/channels.xml

```diff
@@ -110,17 +110,11 @@
 	</channel-type>
 
 	<!-- keypad -->
-	<channel-type id="led">
```

Comment:
Why are you removing this channel?

Reply:
This channel was from a copy/paste when i initially wrote the binding. It is not used.